content/03.results.md

```diff
@@ -43,77 +43,78 @@ Therefore, we jointly considered key genomic features of tumor entities describe
 
 Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR incorrectly entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using current WHO terms (e.g., tumors formerly ascribed primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
 <!--TODO: Update counts below when table is updated-->
-**Table {@tbl:Table1}** lists the subtypes we defined within the OpenPBTA, comprising of LGATs (low grade astrocytic tumors; N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12). <!--SAMPLECOUNT-->
+**Table {@tbl:Table1}** lists the subtypes we defined within the OpenPBTA, comprising of low-grade gliomas (N = 290), high-grade gliomas (N = 141), embryonal tumors (N = 126), ependymomas (N = 30), tumors of sellar region (N = 27), mesenchymal non-meningothelial tumors (N = 11), glialneuronal tumors (N = 10), and chordomas (N = 6). <!--SAMPLECOUNT-->
```

Comment:
For this, we actually will want to update the _table_ to use LGG and HGG terminology, so make the table match the text instead of the other way around.  This will require using the `broad_histology_display` column from the palette to render the table, as we concluded here - https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/268#issuecomment-1125303759